drakrun/drakrun/main.py

```diff
@@ -372,6 +353,38 @@ def process(self):
                 dump_dir = os.path.join(outdir, "dumps")
                 drakmon_log_fp = os.path.join(outdir, "drakmon.log")
 
+                injector_cmd = ["injector",
+                                "-o", "json",
+                                "-d", "vm-{vm_id}".format(vm_id=INSTANCE_ID),
+                                "-r", kernel_profile,
+                                "-i", inject_pid,
+                                "-k", kpgd,
+                                "-m", "writefile",
+                                "-e", f"%USERPROFILE%\\Desktop\\{file_name}",
+                                "-B", os.path.join(workdir, file_name)]
+
+                self.log.info("Running injector...")
+                injector = subprocess.Popen(injector_cmd, stdout=subprocess.PIPE)
+                outs, errs = injector.communicate(b"", 20)
```

Comment:
In entire codebase we're using `subprocess.check_output`, this should do same?

Reply:
`Popen` gives more control over what you do and we do use `Popen` to launch DRAKVUF just a few lines below this

we do use `check_output` mostly for the commands executed in draksetup